### ISSUE TITLE
The lack of breadcrumbs

### ISSUE THREAD

**Issue by t-kudou** [daisy/kb#174]
The lack of breadcrumbs throughout the Knowledge Base makes it difficult to know where in the tree the page you are currently viewing is in your browser.

The current Knowledge Base does not meet the following WCAG requirements listed in this Knowledge Base.

https://kb.daisy.org/publishing/docs/wcag/location.html
G65: Providing a breadcrumb trail

Please consider adding breadcrumbs to all pages.
